h2o-py/h2o/model/metrics_base.py

```diff
@@ -342,7 +349,8 @@ def loglikelihood(self):
         ...                validation_frame = valid)
         >>> pros_glm.loglikelihood()
         """
-        return self._metric_json['loglikelihood']
+        if self._algo == 'glm':
+            return self._metric_json['loglikelihood']
```

Comment:
I'd prefer if you'd not care about the algo. So I'd suggest to use `.get("loglikelihood")` instead of checking the `algo`.

```suggestion
        return self._metric_json.get('loglikelihood')
```

Reply:
I removed the algorithm check per your suggestion.